back/tests/cmd/openbsrv-integration/main.go

```diff
@@ -0,0 +1,324 @@
+package main
```

Comment:
We should go over this in person at the next meeting. It looks great, and I'd like to hear your thoughts about the development process you experienced.

Also, I'd like us to discuss how the testability talk I gave recently might help us keep tests focused and named more clearly (any poor direction at this point is entirely my own doing, so this isn't anything critical toward your work - you've done fantastically).

Reply:
That would be awesome--I should have your slides saved somewhere so I can study and use it for this and future projects. Thanks so much for your detailed review! We have a lot to talk about tomorrow :smile_cat: